db/migrate/20201102160605_split_whodunnit_from_versions.rb

```diff
@@ -0,0 +1,31 @@
+class SplitWhodunnitFromVersions < ActiveRecord::Migration[6.0]
+  def up
+    count = 0
+
+    PaperTrail::Version.all.each do |version|
```

Comment:
You can do the following to enable only loading 1000 into memory at a time :). The table is actually massive so it's possible that we end up loading a ridiculous number of records into memory all at once.

```suggestion
    PaperTrail::Version.all.find_each do |version|
```